Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelSaveTests.swift

```diff
@@ -6,11 +6,13 @@ final class SitterProfileViewModelSaveTests: XCTestCase {
 
     override func setUp() {
         super.setUp()
+        KeyValueStorage.clearStoredSitterData()
```

Comment:
We don't have test cases when sitter is loaded from storage, and we don't imply such possibility. Am I right and is it expected result?

Reply:
We clear the storage in `tearDown()`, later in another test I check that the `sitterIsSet` changes when `save()` is called